1st Year/01. Basic Calculator/calculator.py

```diff
@@ -0,0 +1,35 @@
+import operator
```

Comment:
Do not use inbuilt functions, these programs are for learning purpose, traditional way is preferred.

Reply:
Updated implementation.